fud2/fud-core/src/run.rs

```diff
@@ -104,7 +118,10 @@ pub struct OpEmitData {
     /// The commands run whenever the op run. Each of these must be run in order in a context where
     /// the variables in each cmd are supplied. In particular, this means that variables of the
     /// form "$[i|o]<digit>" are in scope.
-    pub cmds: Vec<String>,
+    ///
+    /// The second and third element of the tuple is a list of the indecies of other commands the given
```

Comment:
```suggestion
    /// The second and third element of the tuple is a list of the indices of other commands the given
```

I am a human spell checker

Reply:
Spelling is hard.
I know there is at least one place where I wrote "indexes" instead of "indices" because I couldn't figure out how to spell the latter.

Solution: set my spell checker to "on" by default instead of off.